ani-cli

```diff
@@ -341,7 +346,6 @@ play_episode() {
     replay="$episode"
     unset episode
     update_history
-    [ "$use_external_menu" = "1" ] && wait
```

Comment:
I think in reverting the mac external menu handling, you accidentally removed this as well.
Otherwise this is looking good